apps/charterafrica/src/components/Postcard/PostCard.js

```diff
@@ -0,0 +1,62 @@
+import { Card, CardContent, CardMedia } from "@mui/material";
+import PropTypes from "prop-types";
+import { forwardRef } from "react";
+
+import LineClampedRichTypography from "../LineClampedRichTypography";
+
+const PostCard = forwardRef((props, ref) => {
+  const { author, date, image, title, sx } = props;
+
+  return (
+    <Card elevation={1} sx={sx} ref={ref}>
+      <CardMedia
```

Comment:
I take it these cards needs to be clickable (use [`CardActionArea`](https://mui.com/material-ui/react-card/#primary-action) and `app/codeforafrica` has a few examples as well).